templates/components/account/order-contents.html

```diff
@@ -1,14 +1,20 @@
-<div>
+{{!-- 
+    @TODO: Remove asap. 
```

Comment:
If the data doesn't come through handlebars or stencil, injecting data with JavaScript using the storefront API is usually how it's done in BigCommerce. I've encountered this issue with multiple other BC projects